src/tcgwars/logic/impl/gen4/DiamondPearlPromos.groovy

```diff
@@ -1018,7 +883,7 @@ public enum DiamondPearlPromos implements LogicCardInfo {
           }
         };
       case ARCEUS_LV_X_DP56:
-        return evolution (this, from:"Arceus", hp:HP120, type:COLORLESS, retreatCost:1) {
+        return levelUp (this, from:"Arceus", hp:HP120, type:COLORLESS, retreatCost:1) {
           pokeBody "Multitype", {
             text "Arceus LV.�s type is the same as its previous Level."
```

Comment:
🤔 encoding issue? should be utf-8

Reply:
Yeah, this and HeartgoldSoulsilverPromos.groovy` need to correct these (or just be regened, since nothing major has been implemented in them yet and neither is in any formats iirc)